maxim/tests/test_groq.py

```
@@ -0,0 +1,445 @@
+            delattr(Maxim, "_instance")
+        
+        if not groqApiKey:
+            self.skipTest("GROQ_API_KEY environment variable is not set")
+
+        self.logger = Maxim().logger()
+        
+        # Initialize Groq client with instrumentation
+        self.client = Groq(api_key=groqApiKey)
+        instrument_groq(self.logger)
+
+    def test_non_streaming_chat_completion(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_non_streaming_chat_completion`

Add return type annotation: `None`

(ANN201)